assets/src/components/UserSettingSnackbar.js

```diff
@@ -13,22 +14,27 @@ function UserSettingSnackbar (props) {
     saved,
     response,
     successMessage = 'Setting saved successfully!',
-    failureMessage = 'Setting not saved.'
+    failureMessage = 'Setting not saved.',
+    debounceAmount = 0
```

Comment:
Can you add a comment the purpose of this variable?

Reply:
same question. Not sure why we need this debounceAmount param.

Reply:
I added a new comment to clarify in the new version of this file. There are other areas of the app that use this snackbar that we don't want to add a debounce to, at least for this issue. For instance, the grade filter setting for the resources accessed page triggers the snackbar. I've left it customizable for the future in case other pages would like a debounce